dplace_app/models.py

```diff
@@ -350,6 +347,10 @@ class Language(models.Model):
     # needs to be null=True because some glottolog languages do not have isocodes
     iso_code = models.ForeignKey('ISOCode', null=True)
     family = models.ForeignKey('LanguageFamily', null=True)
+    
+    @property
+    def societies_count(self):
+        return Society.objects.all().filter(language=self).count()
```

Comment:
If you look at https://github.com/shh-dlce/dplace/commit/4c93cf210d3224eeaf06a822ba1322eecc345f19 you see another alternative: Just use an attribute which can be populated with an appropriate `prefetch_related` call. There shouldn't be too many places in the codebase where db queries are emitted.